PIL/ImageDraw.py

```diff
@@ -305,6 +304,120 @@ def multiline_textsize(self, text, font=None, spacing=4):
             max_width = max(max_width, line_width)
         return max_width, len(lines)*line_spacing
 
+    # Note that font metrics assume an origin on a baseline. So xMin is negative
+    # pixels to the left of the origin, xMax is positive pixels to the right of the
+    # origin, yMin is negative pixels below the baseline, yMax is positive pixels
+    # above the baseline.
+    # if text contains "\n" it is treated as multiple lines of text.
+    #    yMin becomes really negative.
+    #    lineHeight and lineHeightPercent become relevant.
+    #    lineHeightPercent is only used if LineHeight is None, and defaults to 100%.
+    #    Use of a lineHeight < sum( font.getmetrics ) may result in text overlap.
+    #    The best lineHeight would be that returned from the font.font.height
+    #    attribute, but in versions of PIL in which that isn't accessible,
+    #    sum( font.getmetrics()) is used instead.
+    def textInfo( self, text, font=None, lineHeight=None, lineHeightPercent=None ):
+        if font is None:
+            font = self.getfont()
+        lines = text.split('\n')
+        txMax = 0
+        txMin = 0
+        txWid = 0
+        tyMax = None
+        tyMin = None
```

Comment:
What does the `t` indicate in these? Can they be named more descriptively? And let's use width rather than wid.